php-practice_2.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// Q1 tic-tac問題
+for ($i=1; $i < 101; $i++) { 
+  if ($i % 4 === 0 && $i % 5 === 0) {
+    echo 'tic-tac' . "\n";
+  } elseif ($i % 4 === 0) {
+    echo 'tic' . "\n";
+  } elseif ($i % 5 === 0) {
+    echo 'tac' . "\n";
+  } else {
+    echo $i . "\n";
+  }
+}
+
+// Q2 多次元連想配列
+// Q2-1
+$personalInfos = [
+  [
+      'name' => 'Aさん',
+      'mail' => 'aaa@mail.com',
+      'tel'  => '09011112222'
+  ],
+  [
+      'name' => 'Bさん',
+      'mail' => 'bbb@mail.com',
+      'tel'  => '08033334444'
+  ],
+  [
+      'name' => 'Cさん',
+      'mail' => 'ccc@mail.com',
+      'tel'  => '09055556666'
+  ],
+];
+
+echo $personalInfos[1]['name'] . 'の電話番号は' . $personalInfos[1]['tel'] . 'です。' . "\n";
+
+// Q2-2
+foreach ($personalInfos as $index => $infos) {
+  echo $index+1 . '番目の' . $infos['name'] . 'のメールアドレスは' . $infos['mail'] . 'で、電話番号は' . $infos['tel'] . 'です。' . "\n";
```

Comment:
`+`の前後は空白を設けていただきたいです！
```suggestion
  echo $index + 1 . '番目の' . $infos['name'] . 'のメールアドレスは' . $infos['mail'] . 'で、電話番号は' . $infos['tel'] . 'です。' . "\n";
```

Reply:
ご指摘頂いた通り、+の前後に空白を設けました。